solr/core/src/test/org/apache/solr/servlet/TestRequestRateLimiter.java

```diff
@@ -48,7 +47,7 @@ public static void setupCluster() throws Exception {
     configureCluster(1).addConfig(FIRST_COLLECTION, configset("cloud-minimal")).configure();
   }
 
-  @Test
+  @Nightly
```

Comment:
This isn't what I meant, sorry for being unclear. Keep this as `@Test` but when selecting the number of documents and queries do something like https://github.com/apache/lucene-solr/blob/master/lucene/core/src/test/org/apache/lucene/index/TestMultiDocValues.java#L52

Reply:
s/Nightly/Test

Reply:
Updated